RELEASE-NOTES.md

```diff
@@ -1,4 +1,9 @@
 ## RELEASE NOTES
+
+### Version 2.11.0 - December 07 2018
```

Comment:
can you change it to 2.12.0 please and the publish date

Reply:
ahh yes - changed